src/components/text-editor/text-editor.stories.mdx

```diff
@@ -202,7 +202,7 @@ this status.
 With use of `template strings` it is possible to pass multiline validation messages to the component.
 
 <Preview>
-  <Story name="with multiline validation">
+  <Story name="with multiline validation" parameters={{ chromatic: { disable: true }}}>
```

Comment:
why we are disabling this story in `chromatic`?

Reply:
it looks the same as `with validation` snapshot so I think we don't need them